build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Place in root of project and run to build the project and all its tests and artifacts
+FILES="./tests/test_*/*"
```

Comment:
This should be changed to just look under sway-lib-std and sway-lib core

Reply:
It already only looks under `sway-lib-std` as the script is in the root of that sub-folder.
What would we want it to build under `sway-lib-core`?

Reply:
Wait no, the script is in the root of the sway repo!

Reply:
Both `sway-lib-std` & `sway-lib-core` get built for each test project, so I don't think it's necessary to add another step to the script to build `sway-lib-core`.